ROADMAP.md

```diff
@@ -1,102 +1,105 @@
-therefore we generally don’t provide specific target dates for releases.
+The AWS CDK team values feedback and guidance from its community of users, although final decisions on inclusion into the project will be made by AWS. We determine the high-level direction for our open roadmap based on customer feedback and popularity (👍🏽 and comments), security and operational impacts, and business value. Where features don’t meet our goals and longer-term strategy, we will communicate that clearly and openly as quickly as possible with an explanation of why the decision was made.
 
-**Q: Is every feature on the roadmap?**
+## FAQs
 
-A: The AWS Cloud Development Kit roadmap provides transparency on our priority for adding new programming languages,
-developer experience improvements, and service coverage in the AWS Construct Library. The AWS CDK toolkit and AWS
-Construct Library are such a large surface areas we are intentionally keeping the roadmap at a high-level, so not every
-CDK feature request will appear on the roadmap. Instead, the roadmap will include a tracking issue
-for each deliverable that provides a feature overview and contains links to relevant, more granular issues and pull
-requests. If you want to track the status of a specific issue or pull request, you can do so by monitoring that work
-item in the [aws-cdk] GitHub repository.
+**Q: Why did you build an open roadmap?**
+A: Your feedback and suggestions would help in ensuring that we are working on the most important and impactful issues. And if you are making decisions and plans based on what we are developing, this will provide insights on what is coming down the road for the CDK.
 
-**Q: What is a tracking issue?**
+**Q: Why are there no dates on your roadmap?**
+A: Because security and operational stability are our highest priorities, the above new features cannot be provided specific target dates. The roadmap is subject to change at any time, and roadmap issues in this repository do not guarantee a feature will be launched as proposed.
 
-A: We create a tracking issue for each CDK feature, AWS Construct Library module, and jsii-supported programming language. Tracking issues provide a brief summary of the feature and a consolidated view of the work scoped for the release. They include links to design documentation, implementation details, and relevant issues. Tracking issues are living documents that start from a basic template and grow more robust over time as we experiment and learn. You can easily find tracking issues by filtering on the [management/tracking label](https://github.com/aws/aws-cdk/labels/management%2Ftracking).
+**Q: Is everything on the roadmap?**
+A: We will publish high-level direction that is within the scope of the CDK. Minor features and performance improvement tasks are not on the roadmap currently, but we are constantly trying to improve the roadmap so please leave your suggestions [here](https://github.com/aws/aws-sam-cli/issues/3267).
```

Comment:
This appears to be a link to a closed issue on the aws-sam-cli repository, not the CDK. Is that the right place to leave comments and suggestions regarding AWS CDK?